python_modules/libraries/dagster-pyspark/dagster_pyspark/__init__.py

```diff
@@ -2,11 +2,13 @@
 
 from .resources import pyspark_resource
 from .types import DataFrame
+from .utils import struct_type_to_metadata
```

Comment:
this change is not compatible with the currently released 0.14.3 version of dagster